src/Dialogs/Settings/Panels/GaugesConfigPanel.cpp

```diff
@@ -31,6 +32,36 @@ static constexpr StaticEnumChoice final_glide_bar_display_mode_list[] = {
   nullptr
 };
 
+static constexpr StaticEnumChoice flarm_display_location_list[] = {
+  { TrafficSettings::GaugeLocation::AUTO,
+    N_("Auto (follow infoboxes)") },
+  { TrafficSettings::GaugeLocation::TOP_LEFT,
+    N_("Top Left") },
+  { TrafficSettings::GaugeLocation::TOP_RIGHT,
+    N_("Top Right") },
+  { TrafficSettings::GaugeLocation::BOTTOM_LEFT,
+    N_("Bottom Left") },
+  { TrafficSettings::GaugeLocation::BOTTOM_RIGHT,
+    N_("Bottom Right") },
+  { TrafficSettings::GaugeLocation::CENTER_TOP,
+    N_("Center Top") },
+  { TrafficSettings::GaugeLocation::CENTER_BOTTOM,
+    N_("Center Bottom") },
+  { TrafficSettings::GaugeLocation::TOP_LEFT_AVOID_IB,
+    N_("Top Left (Avoid Infoboxes)") },
+  { TrafficSettings::GaugeLocation::TOP_RIGHT_AVOID_IB,
+    N_("Top Right (Avoid Infoboxes)") },
+  { TrafficSettings::GaugeLocation::BOTTOM_LEFT_AVOID_IB,
+    N_("Bottom Left (Avoid Infoboxes)") },
+  { TrafficSettings::GaugeLocation::BOTTOM_RIGHT_AVOID_IB,
+    N_("Bottom Right (Avoid Infoboxes)") },
+  { TrafficSettings::GaugeLocation::CENTER_TOP_AVOID_IB,
+    N_("Center Top (Avoid Infoboxes)") },
+  { TrafficSettings::GaugeLocation::CENTER_BOTTOM_AVOID_IB,
+    N_("Center Bottom (Avoid Infoboxes)") },
```

Comment:
Can you align the Text strings with the one for the Flarm gauge (case etc.) This would mean less translation strings.

Reply:
"Top left". Pretty much all the other strings are written in that fashion.